navigation/index.tsx

```diff
@@ -103,23 +99,21 @@ const ProfileStack = createNativeStackNavigator<ProfileStackParamList>();
 
 const OnboardStack = createNativeStackNavigator<OnboardStackParamList>();
 
-function RootNavigator({ onboard, signedIn }) {
+function RootNavigator({ onboard }) {
   return (
-    <Stack.Navigator>
-      {!onboard && signedIn && (
+    <Stack.Navigator initialRouteName="ProfileOnboard">
```

Comment:
what is this used for? would this cause onboarding screen to get displayed when a user has onboarded

Reply:
yes